plugins/chain_plugin/chain_plugin.cpp

```diff
@@ -212,7 +235,8 @@ void chain_plugin::set_program_options(options_description& cli, options_descrip
                EOS_ASSERT(false, plugin_exception, "");
             }
 #endif
-         }), "Override default WASM runtime")
+         })->default_value(eosio::chain::wasm_interface::vm_type::eos_vm_jit, "eos-vm-jit"), wasm_runtime_opt.c_str()
```

Comment:
I'd like to understand if it's possible to use `chain::config::default_wasm_runtime` here so the default can be defined in a single location

Reply:
used default_wasm_rungime